tethys_gizmos/static/tethys_gizmos/js/cesium_map_view.js

```diff
@@ -299,9 +307,9 @@ var CESIUM_MAP_VIEW = (function() {
                                             orientation : orientation,
                                             model : model,
                                          });
-        entity['layer_name'] = model_data.layer_name
-        entity['layer_variable'] = model_data.layer_variable
-        entity['layer_id'] = model_data.layer_id
```

Comment:
To be consistent with how this is implemented with layers, data attributes would be assigned to a `tethys_data` attribute on the object like so:

```
entity['tethys_data'] = model.data
```

See lines: 231-234. We should also map the legend_* attributes to the object as we do on layers.

Tethys shouldn't care about what is in `data` b/c it is supposed to be used to attach any data you want to the object. The `layer_name`, `layer_variable`, and `layer_id` are ATCore specific data attributes.

Wish I'd caught this early b/c ATCore needs to be updated accordingly...